<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIY飞马交易客户端 V0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt_femas.qrc">
    <normaloff>:/images/ femasImage</normaloff>:/images/ femasImage</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/ backImage);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#aa0000;&quot;&gt;欢迎使用我们的DIY飞马交易客户端&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>退 出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#55007f;&quot;&gt;密 码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>textPassword</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUserID">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入UserID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPassword">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#55007f;&quot;&gt;用户代码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>textUserID</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#55007f;&quot;&gt;投资者编号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>textInvestorID</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="textInvestorID">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入InvestorID</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textUserID</tabstop>
  <tabstop>textInvestorID</tabstop>
  <tabstop>textPassword</tabstop>
  <tabstop>LoginIn</tabstop>
  <tabstop>quit</tabstop>
 </tabstops>
 <resources>
  <include location="qt_femas.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
